sklearn/discriminant_analysis.py

```diff
@@ -25,47 +25,63 @@
 from .utils.extmath import softmax
 from .preprocessing import StandardScaler
 
-
 __all__ = ['LinearDiscriminantAnalysis', 'QuadraticDiscriminantAnalysis']
 
 
-def _cov(X, shrinkage=None):
-    """Estimate covariance matrix (using optional shrinkage).
-
+def _cov(X, shrinkage=None, covariance_estimator=None, standardize=False):
+    """Estimate covariance matrix (using covariance_estimator).
     Parameters
     ----------
     X : array-like, shape (n_samples, n_features)
         Input data.
-
+    covariance_estimator : BaseEstimator
```

Comment:
I'm not sure what to put here, a `BaseEstimator` is definitely not enough. I'd say `estimator, or None, default=None`